tools/clang/test/SemaHLSL/attributes.hlsl

```diff
@@ -855,66 +867,77 @@ bool Test_Loop() {
 // Test ClipPlanes
 float4 ClipPlanesVal;
 [ClipPlanes(ClipPlanesVal)]
+[shader("pixel")]
```

Comment:
I bet this case is why you added the ` && !pSM->IsLib()` to the ClipPlanes check.  ClipPlanes is a vertex attribute, with the additional requirement for the `SV_Position` output.  I think that's the wrong fix.

Reply:
Yeah, the diagnostic was being emitted on certain tests and I didn't understand why, so I just added the attribute. Turns out removing the condition didn't cause the problem to resurface, so I can just resolve this.